src/SFA.DAS.EmployerAccounts.Web/Extensions/AccountDashboardViewModelExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using SFA.DAS.Common.Domain.Types;
+using SFA.DAS.EmployerAccounts.Types.Models.Agreement;
+using SFA.DAS.EmployerAccounts.Web.ViewModels;
+
+namespace SFA.DAS.EmployerAccounts.Web.Extensions
+{
+    public static class AccountDashboardViewModelExtensions
+    {
+        public static bool ShowManageYourLevyLink(this AccountDashboardViewModel model)
+        {
+            if((model.AgreementInfo.Type == AgreementType.NonLevy && model.AgreementInfo.Phase?.Type == PhaseType.EOI))
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        public static bool ShowYourFundingReservationsLink(this AccountDashboardViewModel model)
+        {
+            if(
+                model.ApprenticeshipEmployerType == ApprenticeshipEmployerType.NonLevy || 
```

Comment:
I think this should be AND logic. EmployerType.NonLevy && AgreementType.EOI

Reply:
Perhaps the intent was (in effect) `(NonLevy || NonLevy) && EOI`?

Reply:
There are NonLevy/Levy states on the Employer and the Agreement.  The two are mutually exclusive according to the requirements.  So it is show link if Employer is non levy OR show if Agreement is non levy AND EOI.

Reply:
Ok I think this is fine actually as Non-levy users who have signed a Levy agreement should be able to see both links for finance

Reply:
Updated